common/views/components/SearchFilters/SearchFiltersDesktop.tsx

```diff
@@ -326,6 +323,52 @@ const SearchFiltersDesktop: FunctionComponent<SearchFiltersSharedProps> = ({
     }
   }, [wrapperWidth, hasCalculatedFilters, router.query]);
 
+  return (
+    <>
+      {hasCalculatedFilters ? dynamicFiltersCalculated : dynamicFiltersSource}
+      {dynamicFilters.length < filters.length && (
+        <Space
+          h={{
+            size: 'm',
+            properties: ['padding-left', 'padding-right'],
+          }}
+        >
+          <ButtonSolid
+            colors={themeValues.buttonColors.marbleWhiteCharcoal}
+            icon={filter}
+            isIconAfter
+            hoverUnderline={true}
+            size="small"
+            type={ButtonTypes.button}
+            text="All Filters"
+            clickHandler={event => {
+              event.preventDefault();
+              setShowMoreFiltersModal(true);
+            }}
+            ref={openMoreFiltersButtonRef}
+            isPill
+          />
+        </Space>
+      )}
+    </>
+  );
+};
+
+const SearchFiltersDesktop: FunctionComponent<SearchFiltersSharedProps> = ({
+  query,
+  changeHandler,
+  filters,
+  linkResolver,
+  activeFiltersCount,
+  searchFormId,
+  isNewStyle,
+}: SearchFiltersSharedProps): ReactElement<SearchFiltersSharedProps> => {
+  const [showMoreFiltersModal, setShowMoreFiltersModal] = useState(false);
+  const wrapperRef = useRef<HTMLDivElement>(null);
+  const [componentMounted, setComponentMounted] = useState(false);
+  useEffect(() => setComponentMounted(true));
```

Comment:
This will be fired everytime on each re-render, if it's just for the first load we could add an empty dependency array, makes it work like `componentDidMount`.